### PR TITLE
open accordion by default on non node api services

### PR DESCRIPTION
**Pull request by paulpomerleau**
the specification accordion should now be closed on Node-API and open on all other service endpoint methods.

### CODE REVIEW

File: src/components/APIMethods/APIMethods.tsx [hunk 1]
```diff
@@ -51,12 +53,17 @@ function APIMethod({
   );
 }
 
-export default function APIMethods({ network, methods }) {
+export default function APIMethods({ network, methods, service }) {
+  console.log(service);
```
Comment (pyr0gan) [inline src/components/APIMethods/APIMethods.tsx:57]: Is this logging intentional @paulpomerleau ?

**Review by pyr0gan — COMMENTED** [review]
We may want to replace the text "View specification" with just "Specification" when it's open?
Screen Shot 2022-10-04 at 12 42 34 PM

File: src/components/APIMethods/APIMethods.tsx (hunk 1 above)
Comment (paulpomerleau) [reply src/components/APIMethods/APIMethods.tsx:57]: good catch sorry bout that

Comment (paulpomerleau) [reply src/components/APIMethods/APIMethods.tsx:57]: updated for dynamic title -> good idea

Comment (paulpomerleau) [reply src/components/APIMethods/APIMethods.tsx]: updated the title to be dynamic based on open/close state, good idea

**Review by pyr0gan — APPROVED** [review]

**Pull request merged**